### PR TITLE
Brand management

### PR DESCRIPTION
**Pull request by arijit1111**

### CODE REVIEW

File: Modules/Brand/Http/Controllers/BrandController.php [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+                    'password' => 'required|min:6',
+        ]);
+        if ($validator->fails()) {
+            $response = ['res' => false, 'msg' => $validator->errors()->first(), 'data' => ""];
+        } else {
+            $data = (array) $request->all();
+            $user = User::create([ 'email' => $data['email'], 'first_name' => $data['first_name'], 'last_name' => $data['last_name'], 'password' => Hash::make($data['password']), 'role' => 'brand']);
+            if ($user) {
+
+                //$token = Str::random(64);
```
Comment (shoaibrehman) [inline Modules/Brand/Http/Controllers/BrandController.php]: Remove commented out code.

File: Modules/Brand/Http/Controllers/BrandController.php [hunk 2]
```diff
@@ -0,0 +1,368 @@
...
+            $tmp_arr = explode(".", $file_name);
+            $extension = end($tmp_arr);
+            $original_file_name = pathinfo($file_name, PATHINFO_FILENAME);
+            $file_url = Str::random(10) . '_cstmrs.' . $extension;
+            move_uploaded_file($_FILES["upload_contact_list"]["tmp_name"], $folderPath . $file_url);
+            $vendor->upload_contact_list = $vndr_upload_path . "/" . $file_url;
+            $status = $vendor->save();
+        }
+
+//        if ($vendor->step_count == 12) {
```
Comment (shoaibrehman) [inline Modules/Brand/Http/Controllers/BrandController.php]: Remove commented out code.

File: Modules/Brand/Http/Controllers/BrandController.php [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+        return response()->json($response);
+    }
+
+    /**
+     * Update the specified resource in storage.
+     * @param Request $request
+     * @param int $id
+     * @return Renderable
+     */
+    public function update(Request $request) {
```
Comment (shoaibrehman) [inline Modules/Brand/Http/Controllers/BrandController.php]: This update function is not used anywhere in the project. What's the purpose of this function?

File: Modules/Brand/Http/Controllers/BrandController.php [hunk 4]
```diff
@@ -0,0 +1,368 @@
...
+                $response = ['res' => true, 'msg' => "Successfully updated your account", 'data' => ''];
+            } else {
+                $response = ['res' => false, 'msg' => "Please try again!", 'data' => ''];
+            }
+        }
+
+        return response()->json($response);
+    }
+
+    public function category() {
```
Comment (shoaibrehman) [inline Modules/Brand/Http/Controllers/BrandController.php]: This _category_ function is not used anywhere in the project. What's the purpose of this function?

File: Modules/Brand/Http/Controllers/BrandController.php (hunk 1 above)
Comment (arijit1111) [reply Modules/Brand/Http/Controllers/BrandController.php]: Done

File: Modules/Brand/Http/Controllers/BrandController.php (hunk 2 above)
Comment (arijit1111) [reply Modules/Brand/Http/Controllers/BrandController.php]: Done

File: Modules/Brand/Http/Controllers/BrandController.php (hunk 3 above)
Comment (arijit1111) [reply Modules/Brand/Http/Controllers/BrandController.php]: This function used for account update of brand

File: Modules/Brand/Http/Controllers/BrandController.php (hunk 4 above)
Comment (arijit1111) [reply Modules/Brand/Http/Controllers/BrandController.php]: Deleted this function

**Pull request closed without merging**